blocksync/buffer.go

```diff
@@ -78,7 +79,11 @@ func (b *blockBuffer) Flush(blk *block.Block) (bool, bCheckinResult) {
 		}
 		delete(b.blocks, heightToSync)
 		if err := commitBlock(b.bc, b.ap, b.cs, blk); err != nil && errors.Cause(err) != blockchain.ErrInvalidTipHeight {
-			l.Error("Failed to commit the block.", zap.Error(err), zap.Uint64("syncHeight", heightToSync))
+			if errors.Cause(err) == poll.ErrProposedDelegatesLength || errors.Cause(err) == poll.ErrDelegatesNotAsExpected {
```

Comment:
Can you check if the error you want to ignore actually comes from here instead: https://github.com/iotexproject/iotex-core/blob/8f7700a735449676a2fa8fe87312f02be8138940/consensus/scheme/rolldpos/roundcalculator.go#L215

Reply:
just msg error use errors.Wrapf ,not include a public define